app/rql.py

```diff
@@ -27,6 +27,19 @@ class AuditableMixin(TimestampMixin):
     deleted_by = RelationshipRule(alias="events.deleted.by", rules=ActorRules())
 
 
+class AccountUserRules(ModelRQLRules, AuditableMixin):
+    __model__ = AccountUser
+
+
+class UserRules(ModelRQLRules, AuditableMixin):
+    __model__ = User
+
+    id = FieldRule()
+    email = FieldRule()
+    status = FieldRule()
+    accounts = RelationshipRule(rules=AccountUserRules(), alias="accounts")
```

Comment:
I don't understand what we can filter with this. It will be awesome to be able to issue queries like:

`any(accounts, eq(accounts.account.id,FACC-1234-5678))` but I'm not sure requela already support it.